lib/perl/Genome/Sys.pm

```diff
@@ -1409,6 +1409,12 @@ sub shellcmd {
 
     my ($t1,$t2,$elapsed);
 
+    my @cmdline;
+    if (ref($cmd) and ref($cmd) eq 'ARRAY') {
+        @cmdline = @$cmd;
+        $cmd = join(' ', map $self->quote_for_shell($_), @cmdline));
```

Comment:
Hah, this will be the third time I've removed this paren.  (It came back in a previous rebase I did--doh!)  You're right, the `quote_for_shell` change is important for this.  I will rebase.